src/hb/set_digest.rs

```diff
@@ -0,0 +1,282 @@
+use ttf_parser::GlyphId;
+
+// To make things easier, we don't have the generic parameter mask_t,
+// and assume we always use u32, since this is what is also used in
+// harfbuzz.
+type mask_t = u64;
```

Comment:
The comment says `u32` but it actually uses `u64`; is this a mistake?

Reply:
Oh, yep, thanks! I realized in hindisght that it was `u64` and not `u32`.